tools/wptserve/wptserve/response.py

```diff
@@ -489,13 +490,16 @@ def write_status(self, code, message=None):
                 message = ''
         self.write("%s %d %s\r\n" %
                    (self._response.request.protocol_version, code, message))
+        self._status_written = True
 
     def write_header(self, name, value):
         """Write out a single header for the response.
 
```

Comment:
Document the default status.

Reply:
Oh yes I forgot to add `self.default_status = 200` in `__init__` to make it clearer instead of using magic numbers. Is that enough or do you think I should also add comments to document it?

Reply:
Oh sorry, I meant to request to document the *behaviour* that a default status would be set rather than the *value* of the default status, but `self.default_status` is also great to have.